### PR TITLE
Make names in the ProjectDetails name-value unique

### PR DESCRIPTION
**Pull request by thelee**
Because project details is intended to be pivoted, we can't have duplicate names (at least without a solid way to de-dupe them in powerBI, which would complicate the schema more than just emitting the most authoritative name in the script).

### CODE REVIEW

File: relational/test_table.py [hunk 1]
```diff
@@ -479,112 +499,119 @@ def test_project_details_permit_addenda_summary(basic_graph, d):
...
+                Entry('1',
+                      PPTS.NAME,
+                      [NameValue('residential_units_1br_net', '2', d)]),
+                Entry('2',
+                      MOHCDInclusionary.NAME,
+                      [NameValue('num_2bd_units', '5', d),
+                       NameValue('num_3bd_units', '3', d)]),
+            ],
+            want={
+                'residential_units_1br': '2',
```
Comment (rajpara11) [inline relational/test_table.py]: Not specifically about the test but moreso about the logic, it seems odd to me that we just add up the different data sources here. Wouldn't we want to do what we do with the MOHCD data sources and be internally consistent (so use only one data source)?

Comment (thelee) [reply relational/test_table.py]: you're right. i added the "only one mohcd" logic later on and never updated this part.

Comment (thelee) [reply relational/test_table.py]: that would be a cleaner way to handle the uniqueness aspect

Comment (thelee) [reply relational/test_table.py]: that would be a cleaner way to handle the uniqueness aspect

**Pull request merged**